### PR TITLE
Bug fix 3.5/internal issue #651

### PR DESCRIPTION
**Pull request by gnusi**
PR fixes issues with custom analyzers in a cluster environment

### CODE REVIEW

**Comment by gnusi** [comment]
https://github.com/arangodb/backlog/issues/651

**Comment by gnusi** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/7096/

**Comment by gnusi** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/7097/

**Comment by gnusi** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/7115/

**Comment by gnusi** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/7120/

**Comment by gnusi** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/7121/

File: arangod/IResearch/IResearchLinkMeta.h [hunk 1]
```diff
@@ -126,40 +217,43 @@ struct IResearchLinkMeta {
...
   ///        or (if 'mask' != nullptr) values in 'mask' that are set to false
   ///        elements are appended to an existing object
   ///        return success or set TRI_set_errno(...) and return false
+  /// @param builder output buffer (out-param)
+  /// @param writeAnalyzerDefinition output full analyzer definition instead of just name
+  /// @param ignoreEqual values to ignore if equal
   /// @param defaultVocbase fallback vocbase for analyzer name normalization
   ///                       nullptr == do not normalize
-  /// @param usedAnalyzers add to this map analyzers used in meta,
+  /// @param values to ignore always
```
Comment (Dronplane) [inline arangod/IResearch/IResearchLinkMeta.h]:
```suggestion
  /// @param mask values to ignore always
```

**Comment by gnusi** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/7123/

**Comment by gnusi** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-matrix-pr-windows/1133/

**Comment by KVS85** [comment]
Tests blue.

**Pull request merged**